PSPrintSDK/OLFrameSelectionViewController.m

```diff
@@ -36,26 +36,26 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     vc.assets = self.assets;
     if ([segue.identifier isEqualToString:@"Selected2x2FrameStyleSegue"]) {
         for (OLProduct *product in products){
-            if (product.templateType == kOLTemplateTypeFrame2x2){
+            if (product.productTemplate.quantityPerSheet == 4){
```

Comment:
Is this really a reliable way to identify the correct frame template? Seems like if a new template was created that happened to require the same quantity to fulfil the order this would all fall over? For example where you check quantityPerSheet == 1, would this not match up with a poster template requiring a single image too?